### ISSUE TITLE
Suggestion 

### ISSUE THREAD

**Issue by Misio12320** [Pokechu22/WorldDownloader#94]
Make download world with some like cords from spawn or  0,0 worldborder to the border and after border empty chunks to the end... 🤔🤔🤔 I cant write in pull request So I said request here @Pokechu22

**Comment by Misio12320** [comment]
@Pokechu22 So I need press the chest and will work?

**Comment by Pokechu22** [comment]
Yes, you need to open the chest and then it should save things.  (If you didn't open them this time, then it will have used your old data that was incorrect -- make sure to open all of them).

For your actual request: I think the world border would be useful, but I'm not sure how to make it work perfectly (actually knowing where to place it is hard, especially if it's not a perfect square).  I'll look into it, but I'm not 100% sure about it.  The game _should_ generate empty chunks already, though in the previous release this worked incorrectly.

**Comment by Misio12320** [comment]
Why i get New chunks on empty void... I pasted my builds and after them it just spawning New chunks...
